buildpipeline/DotNet-CoreFx-Trusted-Linux-Crossbuild.json

```diff
@@ -416,7 +416,7 @@
       "value": "/crossrootfs/$(PB_Architecture)",
       "allowOverride": true
     },
-    "portableLinux": {
+    "portableBuild": {
```

Comment:
High-level question: Why rename this property? It is still only used in Linux contexts and AFAIK there's no plan to expand its usage to other operating system families. If we do expand it, we will probably need multiple properties to disambiguate (e.g. `portableGlibcLinux`, `portableMusllibcLinux`, etc.).